lib/salsify_rubocop/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SalsifyRubocop
-  VERSION = '1.27.1'
+  VERSION = '1.3.0'
```

Comment:
As per our [versioning conventions](https://github.com/salsify/salsify_rubocop#versioning) for this gem, this version should be `1.42.0` to match the version of rubocop.

Reply:
Missed that. Apologies.